### PR TITLE
[config:import] return exit code on error (#3083)

### PR DESCRIPTION
**Pull request by cburschka**

### CODE REVIEW

File: src/Command/Config/ImportCommand.php [hunk 1]
```diff
@@ -127,8 +130,9 @@ private function configImport(DrupalStyle $io, StorageComparer $storage_comparer
             $io->success($this->trans('commands.config.import.messages.already-imported'));
         } else {
             try {
-                $config_importer->import();
                 $io->info($this->trans('commands.config.import.messages.importing'));
+                $config_importer->import();
+                return TRUE;
```
Comment (jmolivas) [inline src/Command/Config/ImportCommand.php]: should this be `return 0;` ?

Comment (cburschka-pwc) [inline src/Command/Config/ImportCommand.php]: The `execute()` function is expected to return an exit code (1 on failure, 0 or nothing on success), while the private `configImport()` function is expected to return a success boolean (TRUE on success, FALSE or nothing on failure). So it's weird, but consistent with the current usage...

Comment (jmolivas) [inline src/Command/Config/ImportCommand.php]: My bad, this is a private scope function. not the execute function.

**Comment by jmolivas** [comment]
@cburschka Thanks for your contribution, your PR was merged. This will be included on the next release.

**Pull request merged**